src/main/webapp/app/exam/participate/summary/exercises/programming-exam-summary/programming-exam-summary.component.html

```diff
@@ -19,7 +19,7 @@ <h5>{{ 'artemisApp.exam.examSummary.yourSubmission' | artemisTranslate }}</h5>
                 </div>
                 @if (isAfterResultsArePublished) {
                     <h5>{{ 'artemisApp.exam.examSummary.assessment' | artemisTranslate }}</h5>
-                    @if (feedbackComponentParameters.result) {
+                    @if (feedbackComponentParameters && feedbackComponentParameters.result) {
```

Comment:
Can you also adapt the type to reflect to possible undefined value?
```suggestion
                    @if (feedbackComponentParameters?.result) {
```